data/json/monsters.json

```diff
@@ -843,7 +843,7 @@
     "id": "mon_dog_skeleton",
     "type": "MONSTER",
     "name": "skeletal dog",
-    "description": "This once-canine has shed all its skin revealing a carapace of fused bones and ribs.  This bone armor devoid of external flesh seems to be propelled by a net of sickly black veins and sinews glistening with black goo.",
+    "description": "This once-canine has shed all of its skin, revealing a carapace of fused bones and ribs.  Devoid entirely of flesh, this walking suit of bone seems to be controlled by a net of veins and sinews which pulse with glistening black goo.",
```

Comment:
Suggest `net of veins and sinews which pulse with glistening black goo` -> `network of veins and sinews, which pulse with glistening black goo`.

Reply:
In this context, "net" refers to something like a web; both "net" and "network" are fine, but I think "net" has slightly more oomph behind it.